lib/classes/Variables.test.js

```diff
@@ -42,12 +42,12 @@ describe('Variables', () => {
       const serverless = new Serverless();
 
       const populatePropertyStub = sinon
-        .stub(serverless.variables, 'populateProperty');
+        .stub(serverless.variables, 'populateProperty').returns(Promise.resolve());
```

Comment:
You could do `.resolves()` since we're now using the `sinon-bluebird` package (introduced in #3000)